AUTHORS

```diff
@@ -337,19 +376,25 @@ Wang Li                                 wangli09@kuaishou.com
 Wayne Haber                             whaber@gitlab.com
 Wazir Ahmed                             wazir@accuknox.com
 Weilong Cui                             cuiwl@google.com
+weizhou Lan                             weizhou.lan@daocloud.io
 Wenxian Li                              wofanli@gmail.com
 Will Deuschle                           wdeuschle@palantir.com
 Will Stewart                            will@northflank.com
+wq                                      kyu21@outlook.com
```

Comment:
Looks like someone was just trying to quit vim while writing their signoff.

Reply:
somehow I've missed this person. I've tried to fix most of them.